login/login.go

```diff
@@ -179,14 +184,14 @@ func NewDeviceLogin(args []string) (DeviceLogin, error) {
 	var url string
 	err := Args.Parse(args[1:])
 	if err != nil {
-		return DeviceLogin{}, errors.New("failed parsing arguments")
+		return DeviceLogin{}, err
```

Comment:
As above, passing errors onwards is nice, I generally prefer making a new error (including the old one) to clarify where the current comes from, e.g. `fmt.Errorf("Arguments parsing failed: %v, err)`.